packages/next/src/lib/metadata/types/metadata-types.ts

```diff
@@ -7,8 +7,8 @@
 export interface DeprecatedMetadataFields {
   /**
    * Deprecated options that have a preferred method
-   * @deprecated Use appWebApp to configure apple-mobile-web-app-capable which provides
-   * @see https://www.appsloveworld.com/coding/iphone/11/difference-between-apple-mobile-web-app-capable-and-apple-touch-fullscreen-ipho
+   * @deprecated Use appWebApp to configure mobile-web-app-capable which provides
+   * @see https://www.appsloveworld.com/coding/iphone/11/difference-between-mobile-web-app-capable-and-apple-touch-fullscreen-ipho
```

Comment:
Both old and new reference url have been permanently removed.
So it might be good to delete `* @see https://www.appsloveworld.com/coding/iphone/11/difference-between-mobile-web-app-capable-and-apple-touch-fullscreen-ipho`

```suggestion
   * @deprecated Use appWebApp to configure mobile-web-app-capable which provides
```